service-tests/github.js

```diff
@@ -9,13 +9,53 @@ const {
   isFormattedDate,
   isVPlusDottedVersionAtLeastOne
 } = require('./helpers/validators');
+const colorscheme = require('../lib/colorscheme.json');
+const mapValues = require('lodash.mapvalues');
 
 const t = new ServiceTester({ id: 'github', title: 'Github' });
 module.exports = t;
-
-t.create('License')
-  .get('/license/badges/shields.json')
-  .expectJSONTypes(Joi.object().keys({ name: 'license', value: Joi.string() }));
+const colorsB = mapValues(colorscheme, 'colorB');
+
+t.create('Public domain license')
+  .get('/license/badges/shields.json?style=_shields_test')
+  .expectJSON({ name: 'license', value: 'CC0-1.0', colorB: '#7cd958' });
```

Comment:
Since `licenseToColor` has its own unit tests, how about doing something like:

```js
const permissiveLicenseColor = licenseToColor('CC0-1.0');
```

That way if we change the color, this test won't require new maintenance.

Reply:
Looks like this is pending.

Reply:
Done. I had to stop work on this suddenly. Thanks!